APIDOCS.md

```
@@ -0,0 +1,2055 @@
+    - [Text-to-Speech (GET) 📝➡️🎙️](#text-to-speech-get-️️)
+    - [Text-to-Speech (POST - OpenAI Compatible) 📝➡️🎙️](#text-to-speech-post---openai-compatible-️️)
+  - [MCP Server for AI Assistants 🤖🔧](#mcp-server-for-ai-assistants-)
+  - [React Hooks ⚛️](#react-hooks-️)
+  - [Real-time Feeds API 🔄](#real-time-feeds-api-)
+    - [Image Feed 🖼️📈](#image-feed-️)
+    - [Text Feed 📝📈](#text-feed-)
+  - [Referrer 🔗](#referrer-)
+    - [API Update (starting **2025.03.31**) 📅](#api-update-starting-20250331-)
+    - [Special Bee ✅🐝🍯](#special-bee-)
+  - [Authentication 🔑](#authentication-)
+    - [API Keys and Tokens](#api-keys-and-tokens)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)